keep a history of the past user agents issue #255

haleelsada opened the pull request:
#255 keep a history of the past user agents

AlyaGomaa commented:
hey @haleelsada you don't have to open a new PR every time you add a change 
the code in this PR will  be  automatically updated when you commit to your local haleelsada:issue255  branch

haleelsada commented:
Dear @AlyaGomaa 
I'm really sorry for the inconvenience, the http.log file also changed when I tested the code, and I couldn't undo it at that time hence did a different PR from a different branch, I have fixed it now, you can test this code and close the other PR. Thank you

AlyaGomaa commented:
Hey @haleelsada thank you for your PR!
it's merged.
